src/Extensions/ServiceFabric.Services/ServiceFabricServicesEventSource.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics.Tracing;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.ServiceFabric.Services
+{
+	/// <summary>
+	/// Service Fabric event source
+	/// </summary>
+	[EventSource(Name = "Microsoft-OMEX-Logs-SfServicesInit")] //Breaking Change: Name Changed from "Microsoft-OMEX-Logs"
+	internal sealed class ServiceFabricServicesEventSource : EventSource
+	{
+		/// <summary>
+		/// Instance of service fabric event source
+		/// </summary>
+		public static ServiceFabricServicesEventSource Instance { get; } = new ServiceFabricServicesEventSource();
+
+
+		/// <summary>
+		/// Logs a service type registered ETW event.
+		/// </summary>
+		/// <param name="hostProcessId">Host process id</param>
+		/// <param name="serviceType">Service type</param>
+		[NonEvent]
+		public void LogServiceTypeRegistered(int hostProcessId, string serviceType)
+		{
+			if (!IsEnabled())
+			{
+				return;
+			}
+
+			LogServiceTypeRegistered(hostProcessId, serviceType, $"Service host process {hostProcessId} registered service type {serviceType}");
```

Comment:
>Service [](start = 58, length = 7)

Use `FormattableString.Invariant` as outlined in the other file

Reply:
fixed, thanks